### PR TITLE
Alffd fluff sprites

### PR DESCRIPTION
**Pull request by Allfd**
Adds Alffd AI fluff sprites and hologram.

### CODE REVIEW

**Comment by Dumbdumn5** [comment]
Is skrek.

**Comment by Allfd** [comment]
@Dumbdumn5 

Ya ya, skrek brain is best brain!

**Comment by tigercat2000** [comment]
you could use the new magic hologram stuff to do a full sized armalis probably

**Comment by Allfd** [comment]
@Tigercat2000 

I will take a look tonight,  the sprites in use now are all 32x32 so I scaled it down,  I was not clear on how to do this using the larger sprite file without more code on my part

**Comment by Allfd** [comment]
@tigercat2000 

As per your suggestion, modified to take advantage of the 64 height code that was added for the ancient machine.

Thanks Tigercat!

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -853,7 +853,11 @@ var/list/ai_verbs_default = list(
 				if("ancient machine")
 					holo_icon = getHologramIcon(icon('icons/mob/ancient_machine.dmi', "ancient_machine"))
 				if("custom")
-					holo_icon = getHologramIcon(icon('icons/mob/custom-synthetic.dmi', "[ckey]-ai-holo"))
+					if("[ckey]-ai-holo" in icon_states('icons/mob/custom-synthetic.dmi'))
+						holo_icon = getHologramIcon(icon('icons/mob/custom-synthetic.dmi', "[ckey]-ai-holo"))
+					else
```
Comment (KasparoVy) [inline code/modules/mob/living/silicon/ai/ai.dm]: Any reason why there's no validation to check if("[ckey]-ai-holo" in icon_states('icons/mob/custom-synthetic64.dmi'))?

**Comment by Allfd** [comment]
@KasparoVy 

The custom hologram code will only ever be called if the server owner has put the entry into custom_sprites.txt the purpose of the check is more to determine if it is a 32 or 64 height sprite.

When getHologramIcon is called it generates a sprite before trying to pull the icon.  If no icon is present it returns an empty sprite. 

I am adding a check now to prevent a possible exploit that could occur if non fluff sprite was actually added but the text file was.

**Comment by Fox-McCloud** [comment]
@Necaladun @earthdivine

Tagging you both to make sure Allfd's ckey gets added to the right file once this is patched to live.

**Comment by Allfd** [comment]
@Necaladun @earthdivine
my ckey is "alffd" due to my amazing attention to detail.

**Pull request merged**